services/catalog/src/simcore_service_catalog/api/routes/services.py

```diff
@@ -39,6 +39,7 @@
 }
 
 DIRECTOR_CACHING_TTL = 5 * MINUTE
+LIST_SERVICES_CACHING_TTL = 30
```

Comment:
Question, why not raise this even more? Since this just returns the list of existing services? You could cache for several minutes. It's not like services get released so fast.

Also an endpoint can be added for internal platform usage to `bust` the cache if needed.

Reply:
it's not only new services in the registry, but you can also change the access rights, the names, thumbnail and this kind of stuff. therefore I would not extend this too much.